index.js

```diff
@@ -378,10 +378,7 @@ class ClientServiceBase {
     }
 
     const properties = new ClientProperties(this.properties, [], []);
-    if (
-      properties.getAuditorEnabled() &&
-      properties.getAuditorLinearizableValidationEnabled
-    ) {
+    if (properties.getAuditorEnabled()) {
```

Comment:
When Scalar DL is in the Auditor mode, always execute the validation contract to check the consistency.